src/content/docs/builders/tools/wallets/non-custodial/metakeep.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Metakeep
+description: Intro to Metakeep
+---
+import ToolLayout from "../../../../../../components/Tool/ToolLayout.astro";
+import { Steps } from "@astrojs/starlight/components";
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<ToolLayout
+    introParagraph="MetaKeep user wallets are seamless non-custodial wallets fully owned by end users. Developers can use MetaKeep APIs and Client SDKs to interact with these wallets, obtaining user consent to perform operations."
+    toolName="Metakeep"
+    toolLink="https://docs.metakeep.xyz/docs/create-a-developer-account"
+    toolDocsLink="https://docs.metakeep.xyz/docs/create-a-developer-account"
+>
+
+MetaKeep provides full self-custody of assets without the need for a hardware wallet. They achieve this with a secure, decentralized virtual environment. Their reliable and cost-effective innovations promote mass adoption, and they offer FDIC-inspired insurance policies to enhance user confidence.
```

Comment:
MetaKeep offers API based wallets and functionality through abstracted hardware wallets.
This achieve this through the use of industry standard hardware wallets managed automatically by their service. Their reliable and cost-effective innovations promote mass adoption. Additionally, the team offers insurance polices to enhance user confidence.